eclipse-sirius/sirius-web
# Add support for inter model references

This PR is related to [#1301 Add support for inter model references](https://github.com/eclipse-sirius/sirius-components/issues/1301) issue

# Pull request template

## General purpose
What is the main goal of this pull request?

## Project management

## Architectural decision records (ADR)

## Dependencies

## Backend

This section is not relevant if your contribution does not come with changes to the backend.

### General purpose
- [ ] Is the business code (services) tested? -> Not yet

### Architecture

## Review

### How to test this PR?
* upload a document1 in a project
* create a new document2 in this project
* Add a reference from an object in document1 to an object in document2
* close the project and reopen it
Expected: all the changes are present (it means that save and load worked)
* download document1
Expected: the downloaded document should be the same as the original except the xmi:id and the added reference
